tools/build/pre-installer-diagram-src.xml

```diff
@@ -1 +1 @@
LoMcrr0LWfrBn6hjl++GEf06i3kdlj0tgtG6rcSJ1XPZReutzm1H+rxjSEYdj0kc59NIzuu7CT1wzcEYYs44deSdB96U2xzGp3VYKN/91x5biQJPPek6UOdP61hEy+eeig08486CXYJ1R3t+XXlPzvoggoJ+HpWlC/b76WDWJWSz755DLpjHYivM6TVKgHbNIpyDtXaa993cUXOKDnsR5nOtmyfmeb/AwOyXJ+mBjL+31rNWsLDxLhxp7SHwcIG9keaxpBa3sOuDjv2ZncRnd/q64tdtoI2hM3Fja/s55UWQf/ns4l9jI+N1h/ExpV24sg3AdVtTFEtei5js8cb0qVLZwJFoBo0gQl0w5kl/QvLk40u6jCFlxfhEjHg+pJC9+RK8lnq2P/bGP3Uz/oS6gJq+3g7rohHzWXfOqzMXltezx0O+nDAxiJDQrezxUgNyyZwkR5x1dKFgWFci6V7DBAFYJC74TWP6AUHSR9VfGKoatyiOJRnKkWIdamOIL9eHaMRp1215rIgrUfIFfTY/wKlwMl9RjzcmBPAnQHCiGLhj+4iyFalc8NHDqnQ45i2OUUI09qwfO6Q1yVP3hx4Sm8feNnonwlkKHWCPnVbz0apU81K/z9Uxnq/ES+K7ucv6gPcdzi5G1ZN5X+G30V1rwpIHAkKPhbbPx9nXQea7wXShyrynqryMNYSr+TvY43Sa4nmRjiYOWD9SQ39ckHElAu4VpRCdr5LeOqmlbL4+P2ACvvRU7WAhjRGCreCijyZCmWZkGdPvqecLRTVFhtNYxdVzBlcgrGekH+z+7SRpWdYou9M748sYmrHhuI5ZYwjbj6sljCd2pD2IGxCGOrzX+ACQ8DL9vQlfVNRMNdWL31OpLo7c80zXL5RolgwzMI3aXSSk4BbpP3lPk2Nz5vk6EGRzTiZwQ1+u0A7T05h1iAjC7rk9ILxBaiI/iSMtfPovvYUJZgvWvq9THwRRCwXB8OJbXGcJCbALsQTLEGzVPFL6ohnLElJQmWyKgm5fVzSiJKP0Vvbas4rGntzzYxrVMAvdbqglxtCowunvgOxqA7Wvm5nEFq99DMROcUPb0VSSIjhk5lyoULuoL1ZZ5cGWlnfqylUsSHJDibnnL03/iVJbXpoAV73ojAVIc7Vjmnhodp7oetXtHLzOG20g1EIwh3bQShl5NOrymXiRRbaHb4Pcp57KUKQfSGKDDeb4PDUmz4JnTbxV7oxrvtNc9wT77xTPPce+bljneJ+jW019jihI5l+HifRpzoNFqoegdIXQr2cQKiopz94S+9vTew5S0vVjgorIR7h2vX6UKUixShJpACgySg8yuqLGHLRCjGuseld+XK6besg5JK2fCrfXpLrVzTbPfSOhnXC81Sqh333nYOPpKxwzQPoot4+JyCj4wr9liAfLfSPa6V9t2OlrHGMJIucUtrU/6n93wo/y2tvUW6AWxxIYbCzSluAqsq7CHtsTHAw3sW2kTZtd5GHpZ36Ijy0VBji5BgVUufmZYOVzKQdPoOGNgqgoeFbOExgiW7AmRLqCOhwTyBZIUPNVtY90q4qjHtEwOPLwW3vqniII8A8nT3sSBDRBGJTxtmc7gRiu2Zn+ugeonebTC085KPu0pxNez10NbhKIpCI1qouERDNyA3wChBR/na9jQJx+Yk90ErtwT9BlJfBgwNJiKthxBArCm/TDMtLxm1y2BdJMUq9whagqBcqteF6o8SJ+FsSF6L4V+3G4glgdYSLgF7OlnQz1MbXyCn1xCQXoRRosUNkK986yrb0aX9Av2zRlrB1Rv/0QL8nNHznWmHOe4PU1Ukvo4uj6PWTCdNATsDSnOFvq4Vp1mriXokPDfph+jHf9PdElM53iLfTesbRuIehmkLEXYUHrlRRmt2BUy6rpexOMl391JOleR55JoA9WAYtv/+UCst+pzyMS4Sp4mCrpv9Ye5IwJkhq2CT1BOIGHKeJd3feIIQ1ZgvcqOWQe4+Q1NW6WpZf9MHoHmGFLWEGU/mrah65j0Kr3HvG6CuQyUKwgtwaaw9Sc8g3ppGINBeRa4BxZmzxHEICGnYvk5itHQGAK/bFa4ns8jaM65yKyUR90OzQHnnz0fgoO7BGgbM7ivMCL4kP0D0A3CrehpbCBQGwNBBg14Tob0/lmGJpPCouKtY6zahvNrUWUnJsezna034D5KJFgD4WXvi9QBFbYXHJGNDTtm30qQE03Rl11ggq/WzgGEDHGhUURr/eZFVoDG40Ngk+eeYDyBZd5gQywOoRBXNU4erDDC53gsHBVEiq+564CyC7JDDmygxEu5Z0xwlPZs0amiZtuWnIrxR9/peXqAqsgbZ1I7OLtUGgQs1EGOB2dkOdRkZ5onZ9oq6JT2q8b+6ZollVyOv9xdy6Fsx7qT0VZJuN1vv3R5hP3sUkHRdZZzWkCKmwgbcWsIjDoDbrA4xumM3qR8KxbtL66qRqCI3KxuZuUxV5Y0BtZkvKxN5yUJvAILOwYGnyhegQ8pBmdvqUOVsUiVccOhqYh/fOPO1Cjr5ZMhNqXyJe5CM1pr+E596ijwN6izsPB32ngPyKHrGWpxO/f7C9xpuS20j2W4VJEaWXOW3UW3eUGfcosXZqS8o6bwbH56N/PzDq7Mx3l85Ls3ajJ7YW8ELXPLrWjzvgkD8N9YzfXP9VLPtXwvr+GrjzAHlFYLvf9zd/P/sfz+TjUm6xr7/a0W1ozZyxGeBQo2yu0KLmW//37v///3ZY4rIa4edSJZD8y7rOpMHa3Q3insNKq37Gi2lxsO/XiV40la7P8/PM55M/9g5WEVZsKFZ8EUJv2+pCCnCTG4HjUZH+eYLTMkIv+HPD2cI/imsF3S//Rk29vRB9Nggj3xBaNgyO2xXaJA+xKAmo1mwgyuH37/cb2jzey9V9Pu38Jxtl4220G62cSMo/vU37aT6rIv+cF3NXc1i433NLALHFoV2Y5tmb9edE0Q9N8y1u+jdL59WLjo/ADfiM5Iz4DqxXhhY1p
```

Comment:
Why do you change this file ?

Reply:
@Progi1984 Please ignore this PR, I am trying to figure out how to make PHP 7.4 installable here and shortly I will close this PR.

Reply:
@mvorisek Please update the title and put in "WIP" ;)